consensus/state_processing/src/per_epoch_processing/altair/participation_cache.rs

```diff
@@ -182,7 +182,7 @@ pub struct ParticipationCache {
     /// Caches information about active validators pertaining to `self.previous_epoch`.
     previous_epoch_participation: SingleEpochParticipationCache,
     /// Caches the result of the `get_eligible_validator_indices` function.
-    eligible_indices: Vec<usize>,
+    pub eligible_indices: Vec<usize>,
```

Comment:
```suggestion
    eligible_indices: Vec<usize>,
```

I think this might just be a remnant of an abandoned effort.